components/admin_console/system_users/system_users.jsx

```diff
@@ -4,10 +4,10 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import {FormattedMessage} from 'react-intl';
+import debounce from 'lodash/debounce';
```

Comment:
you can use https://github.com/mattermost/mattermost-redux/blob/ef51b0056f5aa6613827e1475cde1339320aeef2/src/actions/helpers.js#L110 as `import {debounce}  from 'mattermost-redux/actions/helpers`

Reply:
Oh, thanks